autokeras/auto/auto_pipeline.py

```diff
@@ -1,29 +1,30 @@
-class AutoPipeline:
+from autokeras.auto import auto_model
+
+
+class AutoPipeline(auto_model.AutoModel):
     """An AutoModel plus preprocessing and postprocessing.
 
     The preprocessing can include encoding, normalization, and augmentation.
     The postprocessing can include decode the labels from one-hot encoding.
     """
 
-    def fit(self, x_train, y_train, **kwargs):
-        """Tuning the model.
-
-        Arguments:
-            x_train: An instance of numpy.ndarray.
-            y_train: An instance of numpy.ndarray.
-        """
+    def fit(self, **kwargs):
+        """Tuning the model. """
         pass
 
     def predict(self, x_test, postprocessing=True):
         """Predict the output for a given testing data.
 
         Arguments:
-            x_test: The x_test should be a numpy.ndarray.
+            x_test: An instance compatible for input to a Keras Model.
```

Comment:
"Data in a format that can be passed to a Keras model: either a Numpy array, a Python generator of arrays, or a TensorFlow Dataset.`